db/db_test.go

```diff
@@ -50,7 +52,23 @@ func DBGetSSLOptions() options.SSL {
 	}
 }
 
-func TestNewSessionProvider(t *testing.T) {
+func DBGetConnString() *options.URI {
+	if testtype.HasTestType(testtype.SSLTestType) {
+		return &options.URI{
+			//ConnectionString: "mongodb://localhost" + DefaultTestPort + "/",
+			ConnString: connstring.ConnString{
+				SSLCaFileSet:                   true,
+				SSLCaFile:                      "../db/testdata/ca-ia.pem",
+				SSLClientCertificateKeyFileSet: true,
+				SSLClientCertificateKeyFile:    "../db/testdata/test-client.pem",
+			},
+		}
+	}
+
+	return &options.URI{}
+}
+
+func r(t *testing.T) {
```

Comment:
nit: Looks like this used to be called `TestNewSessionProvider` but was replaced by `r`. Is that intentional?

Reply:
seems like it's probably not @huan-Mongo

Reply:
Ah, this is by accident. Thanks for finding out. Fixed!